recipes/jupyterlab-tour/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jupyterlab-tour" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a1fb867d5278f27dc43ffd8a976788e5a6fff769c7f482aed5f6c9aace9b47a1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - nodejs >=12.0.0
+  host:
+    - python >=3.6
+    - pip
+    - jupyterlab >=3.0.0
+    - jupyter-packaging >=0.7.9,<0.8.0
+  run:
+    - python >=3.6
+    - jupyterlab >=3.0.0
+
+test:
+  requires:
+    - nodejs >=12.0.0
+  commands:
+    - jupyter labextension list 2>&1 | grep -ie "jupyterlab-tour.*OK"
+    - python -m jupyterlab.browser_check  # [not linux]
```

Comment:
i don't think browser_check adds much value, and can break for weird reasons unrelated to the system-under-build (e.g. build container doesn't have system x11 drivers, etc). If we're shipping the stuff directly from the sdist, just going with the labextension check is probably sufficient, and more likely to catch packaging errors vs nodejs/browser/driver errors.

```yaml
        - jupyter labextension list
        - jupyter labextension list 1>labextensions 2>&1
        - cat labextensions | grep "jupyterlab-tour.*OK.*jupyterlab_tour"  # [unix]
```